packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -475,6 +481,10 @@ export const BRIDGE_ENCODERS: {
     ]),
     [ERC20BridgeSource.KyberDmm]: AbiEncoder.create('(address,address[],address[])'),
     [ERC20BridgeSource.Lido]: AbiEncoder.create('(address)'),
+    [ERC20BridgeSource.Clipper]: AbiEncoder.create([
+        { name: 'provider', type: 'address' },
+        { name: 'data', type: 'bytes' },
```

Comment:
If this is always empty why encode it at all?

Reply:
they want us to send some affiliate data in here, working it out now